### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by stuts123**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,4 +108,4 @@ We'd love to hear from you no matter what! Email us at either
 or 
 [brianslho@gmail.com](mailto:brianslho@gmail.com)
 or
-[stuts123456789@gmail.com](mailto:stuts123456789@gmail.com)!
\ No newline at end of file
+[stuts123456789@gmail.com](mailto:stuts123456789@gmail.com)!
```
Comment (twbrianho) [inline README.md:111]: Wait, how has this line changed, I can't see the difference.

File: README.md [hunk 2]
```diff
@@ -99,7 +99,7 @@ exit
 ## Contact Me
 Have questions?
 Got a puzzle you want to share?
-Want to criticize my terrible code?
+Want to criticize Brian's terrible code or praise LJ's wonderful code?
```
Comment (twbrianho) [inline README.md:102]: Joke's on you my imposter syndrome wholeheartedly agrees with this.

**Review by twbrianho — APPROVED** [review]
Go ahead and merge 😞

File: README.md (hunk 2 above)
Comment (stuts123) [reply README.md:102]: boo

**Pull request closed without merging**